### PR TITLE
Fixed issues in Production environment

### PR DESCRIPTION
**Pull request by Ananthavijay**
Fixes https://github.com/CircuitVerse/CircuitVerse/issues/1084

#### Describe the changes you have made in this PR -
- Uglifier reported errors because at some places ES6 syntax was used, so converted ES6 syntax to ES5 by using https://babeljs.io/
- Configured Rack deflater, Source: https://artsy.github.io/blog/2012/02/24/10x-rack-and-rails-output-compression-with-rack-deflater/

### CODE REVIEW

**Comment by Ananthavijay** [comment]
@tachyons @satu0king  Please review

File: config/application.rb [hunk 1]
```diff
@@ -18,6 +18,9 @@ class Application < Rails::Application
     # configuring mailer previews directory
     config.action_mailer.preview_path = "#{Rails.root}/lib/mailer_previews"
 
+    config.middleware.use Rack::Deflater
```
Comment (satu0king) [inline config/application.rb]: We are no longer going to be using this

Comment (satu0king) [reply config/application.rb]: If it works in production, I guess its fine.

**Comment by Ananthavijay** [comment]
The code climate issues suggests to use ES6 syntax instead of ES5

**Pull request merged**